src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj

```diff
@@ -15,9 +16,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Http.Connections.Common" />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
     <Reference Include="Microsoft.Extensions.Options" />
+    <Reference Include="System.Text.Encodings.Web" />
```

Comment:
Shouldn't need to add this

Reply:
Looks like you're right - thought I remembered this breaking without it, but looks like it builds fine without the reference